src/metrics/miscellaneousMetrics.ts

```diff
@@ -0,0 +1,31 @@
+import { PromMetricGauge } from '../clients/victoriaMetrics/promMetricGauge';
+import { allSockets } from '../sockets/sockets';
+import { PromMetricCounter } from '../clients/victoriaMetrics/promMetricCounter';
+
+const onlinePlayersMetric = new PromMetricGauge({
+  name: `online_players_total`,
+  help: `Number of online players.`,
+  collect() {
+    this.set(allSockets.length);
+  },
+});
+
+export const uniqueLoginsMetric = new PromMetricCounter({
+  name: 'unique_logins_total',
+  help: 'Total number of unique logins over a 24h time period.',
+});
+
+export const avatarSubmissionsMetric = new PromMetricCounter({
```

Comment:
Can we add a metric counter for approvals as well? 
May as well add rejects as well.